Test are failing on Mac OSX Catalina

```
_______________________ test_process_isolation_settings ________________________

    def test_process_isolation_settings():
        rc = RunnerConfig('/')
        rc.artifact_dir = '/tmp/artifacts'
        rc.playbook = 'main.yaml'
        rc.command = 'ansible-playbook'
        rc.process_isolation = True
        rc.process_isolation_executable = 'not_bwrap'
        rc.process_isolation_hide_paths = ['/home', '/var']
        rc.process_isolation_show_paths = ['/usr']
        rc.process_isolation_ro_paths = ['/venv']
        rc.process_isolation_path = '/tmp'

        with patch('os.path.exists') as path_exists:
            path_exists.return_value=True
            rc.prepare()

        assert rc.command[0:8] == [
            'not_bwrap',
            '--die-with-parent',
            '--unshare-pid',
            '--dev-bind', '/', '/',
            '--proc', '/proc',
        ]

        # hide /home
        assert rc.command[8] == '--bind'
        assert 'ansible_runner_pi' in rc.command[9]
>       assert rc.command[10] == '/home'
E       AssertionError: assert '/System/Volumes/Data/home' == '/home'
E         - /System/Volumes/Data/home
E         + /home

test/unit/test_runner_config.py:498: AssertionError
```

I believe that it's failing because `/home` on Catalina is a symbolic link to `/System/Volumes/Data/home`.

```
lrwxr-xr-x  1 root  wheel  25 Jan 24 14:44 /home -> /System/Volumes/Data/home
```

--- a/pytest.ini
+++ b/pytest.ini
@@ -1,0 +1,4 @@
+[pytest]
+markers =
+    serial: tests that cannot be reliably ran with pytest multiprocessing
+    timeout: used with pytest-timeout

--- a/test/unit/test_runner_config.py
+++ b/test/unit/test_runner_config.py
@@ -525,7 +525,7 @@ def test_process_isolation_settings():
     # hide /home
     assert rc.command[8] == '--bind'
     assert 'ansible_runner_pi' in rc.command[9]
-    assert rc.command[10] == '/home'
+    assert rc.command[10] == os.path.realpath('/home')  # needed for Mac
 
     # hide /var
     assert rc.command[11] == '--bind'

